src/odemis/gui/cont/tabs/__init__.py

```diff
@@ -0,0 +1,68 @@
+# -*- coding: utf-8 -*-
+
+"""
+@author: Rinze de Laat, Éric Piel, Philip Winkler, Victoria Mavrikopoulou,
+         Anders Muskens, Bassim Lazem
+
+Copyright © 2012-2022 Rinze de Laat, Éric Piel, Delmic
+
+Handles the switch of the content of the main GUI tabs.
+
+This file is part of Odemis.
+
+Odemis is free software: you can redistribute it and/or modify it under the
+terms of the GNU General Public License version 2 as published by the Free
+Software Foundation.
+
+Odemis is distributed in the hope that it will be useful, but WITHOUT ANY
+WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+PARTICULAR PURPOSE. See the GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License along with
+Odemis. If not, see http://www.gnu.org/licenses/.
+
+"""
+
+from odemis.gui.model import TOOL_ZOOM, TOOL_ROI, TOOL_ROA, TOOL_RO_ANCHOR, \
+    TOOL_POINT, TOOL_LINE, TOOL_SPOT, TOOL_ACT_ZOOM_FIT, TOOL_RULER, TOOL_LABEL, \
+    TOOL_FEATURE
+
+# The constant order of the toolbar buttons
+TOOL_ORDER = (TOOL_ZOOM, TOOL_ROI, TOOL_ROA, TOOL_RO_ANCHOR, TOOL_RULER, TOOL_POINT,
+              TOOL_LABEL, TOOL_LINE, TOOL_SPOT, TOOL_ACT_ZOOM_FIT, TOOL_FEATURE)
+
```

Comment:
I'd suggest to move these constants to `odemis.gui.__init__.py` (also the part which is in `gui.model.__init__.py`).

Reply:
Can I move the `TOOL_ORDER` constant to `gui.model.__init__.py` instead?

Reply:
You could. I find it a little bit less matching because this is about the presentation (order) and the model should not really care about that. That's why I would suggest to just move everything to gui.init.py . But if you want to change this refactoring light, it's fine to move to model, and later we could move it again...

Reply:
> But if you want to change this refactoring light, it's fine to move to model, and later we could move it again...

Indeed I want to keep this refactoring light.